### ISSUE TITLE
Reload not working

### ISSUE THREAD

**Issue by andreanadnomix** [dnwjn/nova-button#49]
Hi 

**What**
-`->reload()` does not seem to work

![reload](https://github.com/dnwjn/nova-button/assets/143768926/e79e8a5b-11c8-4aa6-a1db-fd4dc9909142)

This is my code
```
Button::make($toggleText)
                ->event(MyEvent::class)
                ->reload()
                ->indexName('Toggle Status')
                ->style('primary')
                ->classes(['text-white'])

```

I verified that there's no error on the event

**Comment by dnwjn** [comment]
Hi @andreanadnomix! Thank you for reporting this bug. I've identified the issue and I'm pushing a fix right away!

**Comment by dnwjn** [comment]
@andreanadnomix This is fixed in v4.3.2!

**Comment by andreanadnomix** [comment]
Thanks bro @dnwjn
